ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -42,82 +42,67 @@ $ sudo -i
 
 This will give you access as an Admin/Superuser/Root.
 
-**NOTE 1:** After the command ``sudo -i`` the computer will ask you for a password, the password is the same you are using for your login. Also note that it will not show your password when typed, it will instead keep a blank field while typing, so type it carefully and press enter.
+**NOTE 1:** After the command `sudo -i` the computer will ask you for a password, the password is the same you are using for your login. Also note that it will not show your password when typed, it will instead keep a blank field while typing, so type it carefully and press enter.
 
-**NOTE 2:** you can distinguish between Admin and User in the Terminal (and also in this document). A user has a `$-sign` when you are logged in a user, and `#-sign` if you are**
+**NOTE 2:** you can distinguish between Admin and User in the Terminal (and also in this document). A user has a `$-sign` when you are logged in a user, and `#-sign` if you are logged in as admin.
 
-```
-# apt-get update
-```
+- **`# apt-get update`**
 
 This will give you the latest available package versions in the repositiory.
 
 When it is done with getting the latest package version, you will need to download and install them. Then...
 
-```
-# apt-get upgrade
-```
+- **`# apt-get upgrade`**
 
 This will download and Install the updates, this might take some time to complete. **NOTE:** it will ask you at the end if you want to continue, for this type **"y"** and press enter.
 
-
-
 ## Stage three 
 
 While you are still logged in as Admin/Superuser/ROOT you will need to add the repository.
 
 Type following:
-```
-nano /etc/apt/sources.list
-```
+
+- **`# nano /etc/apt/sources.list`**
 
 Head to the bottom and press enter to insert new line and type following:
 
-```
-deb http://packages.arctica-project.org/ubuntu-nightly xenial main
-```
+- **`# deb http://packages.arctica-project.org/ubuntu-nightly xenial main`**
```

Comment:
same here, of course this is not a shell command, so "#" does not make sense here!!! Remote "'#" again.